airbyte-integrations/connectors/source-aws-cloudtrail/source_aws_cloudtrail/spec.json

```diff
@@ -10,7 +10,7 @@
       "aws_region_name",
       "start_date"
     ],
-    "additionalProperties": false,
+    "additionalProperties": true,
```

Comment:
what are the additional fields? should we add them to the schema?

Reply:
I have optional config to limit amount of records to read. this allows to speedup test execution

more details here - https://github.com/airbytehq/airbyte/pull/4122#issuecomment-863469344